bundles/org.openhab.ui/web/src/components/addons/addon-card.vue

```diff
@@ -1,45 +1,57 @@
 <template>
-  <f7-link
-    v-if="addon"
-    class="addon-card padding-right-half" :href="addon.id">
-    <div class="addon-card-headline">
-      <div>{{ headline || autoHeadline }}</div>
-    </div>
-    <div class="addon-card-title">
-      <div class="addon-card-title-after">
-        <f7-preloader v-if="addon.pending" color="blue" />
-        <f7-button v-else-if="addon.installed" class="install-button prevent-active-state-propagation" text="Remove" color="red" round small @click="buttonClicked" />
-        <f7-button v-else class="install-button prevent-active-state-propagation" :text="installActionText || 'Install'" color="blue" round small @click="buttonClicked" />
-      </div>
-      <div class="addon-card-label">
-        {{ addon.label }}
+  <f7-link v-if="addon" class="addon-card padding-right-half" :href="addon.id">
+    <div class="addon-card-inner card-expandable">
```

Comment:
Those cards will expand and going back will bug unless you add the `card-prevent-open` class as well.

Like I said IMO the other/better alternative is add `background-color var(--f7-list-link-hover-bg-color)` it on `.addon-card:hover`, without any border, so that the look of these "cards" is the same as the list items - it solves your initial issue (highlighting the image + info text) at least on desktops and remains consistent across designs.

Reply:
I will take a look at it when I am back home (a few days). The reason i used the card-expand class is that it fits the design of other cards. For example, there seems to be some box shadow on the card class

Reply:
I changed card-expandable to just card now, I don't know anymore why I used exandable cards in the first place. 

> Like I said IMO the other/better alternative is add background-color var(--f7-list-link-hover-bg-color) it on .addon-card:hover

You mean instead of cards at all so only the currently hovered addon is highlighted by background color? As you already wrote, this would only fix it on PC and I feel like it wouldn't be the best UX if you can only tell distinguish the current card from the others but not those apart

> so that the look of these "cards" is the same as the list items 

![image](https://user-images.githubusercontent.com/46248939/197357303-a1713102-fd74-4f49-8058-e85448d0e102.png)

This is how it looks combined with list items. I think it looks okay and it looks like the cards are meant to promote some featured addons when combined with list items, so is it that bad when they look a bit different?